Sources/LiveViewNative/Property Wrappers/LiveBinding.swift

```diff
@@ -162,7 +162,6 @@ public struct LiveBinding<Value: Codable> {
         nonmutating set {
             // update the local value
             data.mode = .local(newValue)
-            data.objectWillChange.send()
```

Comment:
This was necessary to get the menu-style picker to update when a menu option is tapped. Is it causing a problem elsewhere?

Reply:
I was receiving runtime warnings about modifying state during a view update from this line at one point, but I can't replicate it anymore.

Reply:
Hmm, that warning would make sense, but I'm not able to reproduce it either. We'll have to keep an eye out for it :/